<ui version="4.0" >
 <class>Location</class>
 <widget class="QDialog" name="Location" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>301</width>
    <height>540</height>
   </rect>
  </property>
  <property name="minimumSize" >
   <size>
    <width>301</width>
    <height>450</height>
   </size>
  </property>
  <property name="maximumSize" >
   <size>
    <width>301</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle" >
   <string>Qsalat - location</string>
  </property>
  <widget class="QLabel" name="label" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>40</y>
     <width>54</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text" >
    <string>Location</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="locationLineEdit" >
   <property name="geometry" >
    <rect>
     <x>60</x>
     <y>40</y>
     <width>161</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="searchButton" >
   <property name="geometry" >
    <rect>
     <x>230</x>
     <y>40</y>
     <width>61</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text" >
    <string>Search</string>
   </property>
  </widget>
  <widget class="QWebView" name="webView" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>79</y>
     <width>281</width>
     <height>281</height>
    </rect>
   </property>
   <property name="url" >
    <url>
     <string>http://www.skanderjabouzi.com/qpray/</string>
    </url>
   </property>
  </widget>
  <widget class="QLabel" name="latLabel" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>380</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text" >
    <string>Latitude</string>
   </property>
  </widget>
  <widget class="QLabel" name="lngLabel" >
   <property name="geometry" >
    <rect>
     <x>140</x>
     <y>380</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="statusTip" >
    <string>Longitude</string>
   </property>
   <property name="text" >
    <string>Longitude</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelButton" >
   <property name="geometry" >
    <rect>
     <x>219</x>
     <y>500</y>
     <width>71</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text" >
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="applyButton" >
   <property name="enabled" >
    <bool>true</bool>
   </property>
   <property name="geometry" >
    <rect>
     <x>90</x>
     <y>500</y>
     <width>121</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text" >
    <string>Show coordinates</string>
   </property>
  </widget>
  <widget class="QPushButton" name="okButton" >
   <property name="geometry" >
    <rect>
     <x>20</x>
     <y>500</y>
     <width>61</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text" >
    <string>Save</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>10</y>
     <width>281</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font" >
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text" >
    <string>Set your location</string>
   </property>
   <property name="alignment" >
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="countryLabel" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>410</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="statusTip" >
    <string>Longitude</string>
   </property>
   <property name="text" >
    <string>Country</string>
   </property>
  </widget>
  <widget class="QLabel" name="cityLabel" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>440</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="statusTip" >
    <string>Longitude</string>
   </property>
   <property name="text" >
    <string>City</string>
   </property>
  </widget>
  <widget class="QLabel" name="timeZoneLabel" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>470</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="statusTip" >
    <string>Longitude</string>
   </property>
   <property name="text" >
    <string>Timezone</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="latLineEdit" >
   <property name="geometry" >
    <rect>
     <x>70</x>
     <y>380</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lngLineEdit" >
   <property name="geometry" >
    <rect>
     <x>210</x>
     <y>380</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="countryLineEdit" >
   <property name="geometry" >
    <rect>
     <x>70</x>
     <y>410</y>
     <width>201</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="cityLineEdit" >
   <property name="geometry" >
    <rect>
     <x>70</x>
     <y>440</y>
     <width>201</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="timezoneLineEdit" >
   <property name="geometry" >
    <rect>
     <x>70</x>
     <y>470</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QWebView</class>
   <extends>QWidget</extends>
   <header>QtWebKit/QWebView</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
